scripts/find_new_dumps.py

```diff
@@ -86,17 +89,33 @@ def get_info_dict(input_dir: Optional[Path]) -> dict:  # Optinal[path]
     return out_dict
 
 
-def help_func():
+def help_func(url: str, input_dir: Optional[Path]):
+    if input_dir is not None:
+        ind_in_dir = {path.name for path in input_dir.iterdir()}
+    else:
+        ind_in_dir = set()
     webpage = requests.get('https://commoncrawl.org/the-data/get-started/')
-    soup = bs4.BeautifulSoup(webpage.text, features='html5lib')
-    for ul in soup.find_all('ul'):
-        if 'CC-MAIN' in ul.text:
-            for idx, li in enumerate(ul.findChildren('li')):
-                info = get_info_from_line(li)
-                if info is not None:
-                    date_str, index_name = info
-                    print(date_str, index_name)
-            break
+    soup = bs4.BeautifulSoup(webpage.text)
+    if url is not None:
+        for ul in soup.find_all('ul'):
+            if 'CC-MAIN' in ul.text:
+                for idx, li in enumerate(ul.findChildren('li')):
+                    info = get_info_from_line(li)
+                    if info is not None:
+                        date_str, index_name = info
+                        print(date_str, index_name)
+                break
```

Comment:
Ez az egész rész ugyanaz, mint amit a `get_info_dict()` visszaad. Pont azért használunk függvényeket, hogy az ilyen felesleges és veszélyes kódismétléseket elkerüljük. Ezt az egészet töröld ki, és a függvény bemeneti paramétere legyen az `info_dict`, ugyanúgy, mint a `send_dict_to_url`-nél, és a `print` is az `info_dict` végigolvasásával menjen.